File: pset6/mario/mario.py
# import get_int function
from cs50 import get_int

# set height as 0
height = 0

# Prompt int
while not (height >= 1 and height <= 8):
    height = get_int("Height: ")

# Draw pyramid


def pyramid(h):
    tmp = height
    if h == 0:
        return
    else:
        pyramid(h-1)
        for i in range(tmp - h):
            print(" ", end="")
        for i in range(h):
            print("#", end="")
        print()


# Print pyramid
pyramid(height)

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < ((height * 2) + 2); j++)
        {
            if (((j < height) && (height <= (i + j + 1))) || ((j > height + 1) && ((height > (j - i - 2))))) 
            {
                printf("#");
            }
            else if (height == (j - i - 2))
            {
                printf("#");
                break;
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}